### ISSUE TITLE
Cannot install this in another npm-based project because the cli.sh file tries to write in non-existing locations

### ISSUE THREAD

**Issue by PissedCapslock** [luciad/typescript-documentation#28]
What I did:

- Clone this repo
- Run `npm pack` in this repository which creates a tar ball 
- Run `npm install path/to/tarball` in my actual project
- Call `npx l-td -i ...` in my actual project

```
      [npm] Error: Command failed: npx l-td -i /home/rstevens/development/2020.1/product/LuciadRIA/release/doc/tsdoc/documentation.json -o /home/rstevens/development/2020.1/product/LuciadRIA/release/doc/tsdoc -m /home/rstevens/development/2020.1/product/LuciadRIA/release/doc/tsdocprep/images
      [npm] cp: cannot create regular file 'content/docu.json': No such file or directory
```

I think the problem is that the script assumes it will be called from the directory containing the script, which is not the case.
While this can easily be solved (e.g. https://stackoverflow.com/a/246128/1076463), you might want to rethink this whole approach.

Assume somebody installs this globally and runs 2 builds in parallel. Both builds will try to overwrite that content/docu.json file. I think the same applies to other stuff you copy-paste (like the themes, ...).

Perhaps it's an option to copy-paste that into the output dir instead ? You know you will have write access to that directory.

**Comment by TijlB99** [comment]
Directories now no longer are relative. About the global part: I think gatsby would still use the dir for its own cache, so I'm not sure how to fix that yet, I'll make a new issue for it.
